engine/spec/components/header_spec.rb

```diff
@@ -0,0 +1,158 @@
+# frozen_string_literal: true
+
+RSpec.describe CitizensAdviceComponents::Header, type: :component do
+  context "with no slots" do
+    subject(:component) do
+      render_inline(described_class.new)
+    end
+
+    it "does not render" do
+      expect(component.at("header")).not_to be_present
+    end
+  end
+
+  context "with logo" do
+    subject(:component) do
```

Comment:
Using subject will change the value of the `subject` invocation. It also gives you access to a shorthand one liner.

i.e. you can say `it { is_expected.to be_present }` When you have a defined subject.

Probably best to alter this one to be `let`